docs/alternator/compatibility.md

```diff
@@ -286,3 +286,7 @@ they should be easy to detect. Here is a list of these unimplemented features:
 * Alternator does not support the new DynamoDB feature "export to S3",
   and its operations DescribeExport, ExportTableToPointInTime, ListExports.
   <https://github.com/scylladb/scylla/issues/8789>
+
+* Alternator does not support the new DynamoDB feature "import from S3",
```

Comment:
I would not use "new", as this doc might live for some time.
Better to write something like added in [Aug 2022](https://aws.amazon.com/blogs/database/amazon-dynamodb-can-now-import-amazon-s3-data-into-a-new-table/)